### ISSUE TITLE
The large image for a post type (seen when you add a new post) is not customized

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#136]
```
If you create a post type that uses a custom menu icon, the icon shows up 
*only* in the left-hand menu, NOT in at the top of the page when you edit or 
create a post in that post-type.  It'd be better UI to have that top icon 
changeable.  

So all the 16x16 custom menu icons that are available may require a larger 
counterpart.  To further complicate things, the icon is controlled via CSS:

{{{
<div id="icon-edit" class="icon32 icon32-posts-tooltip">
}}}

So to fully integrate this, we may have to filter the admin head and add our 
own style declarations that reference the 32x32 versions of the icons.
```

Original issue reported on code.google.com by `fireproofsocks` on 25 Jul 2011 at 9:15

**Comment by GoogleCodeExporter** [comment]
```
You should be able to override the style:

#icon-edit, #icon-post {
  background-attachment:initial;
  background-clip:initial;
  background-color:transparent;
  background-image:url(http://yoursite.com/wp-admin/images/icons33.png?ver=20100531);
  background-origin:initial;
  background-position:-552px -5px;
  background-repeat:no-repeat no-repeat;
}

But this will have to require that each 16x16 post_type image will have a 32x32 
counterpart. See issue 30 
(http://code.google.com/p/wordpress-custom-content-type-manager/issues/detail?id
=30).
```

Original comment by `fireproofsocks` on 15 Aug 2011 at 3:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `fireproofsocks` on 3 Sep 2011 at 4:09
- Added labels: **Milestone-Release0.9.4**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fireproofsocks` on 16 Sep 2011 at 5:24
- Added labels: **Milestone-Release0.9.5**
- Removed labels: **Milestone-Release0.9.4**

**Comment by GoogleCodeExporter** [comment]
```
Ok, this has been implemented in 0.9.4.

Committed revision 440620.
```

Original comment by `fireproofsocks` on 20 Sep 2011 at 5:32
- Changed state: **Fixed**
- Added labels: **Milestone-Release0.9.4**
- Removed labels: **Milestone-Release0.9.5**
